### ISSUE TITLE
Pre-work Checklist: Developer: Kazushi Rickert

### ISSUE THREAD

**Issue by plang-psm** [hackforla/website#3882]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [ ] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by KazushiR** [comment]
Provide Update
1. Progress: Working on first issue.
2. Blockers: None
3. Availability: flexible
4. ETA: sometime this week.

**Comment by KazushiR** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?

It took me about 30 minutes to complete my first issue. My avalibility was pretty flexible during the completion of my first issue.

**Comment by ExperimentsInHonesty** [comment]
@KazushiR I am going to reopen this issue until you have been on an issue over a week, and have done the progress update 
Provide Update
1. Progress
2. Blockers
3. Availability
4. ETA
without being asked by another teammate.  It may  not end up not being necessary (time wise) until you get to a large issue.

**Comment by KazushiR** [comment]
Progress: This has been compelted.
Blockers: No blockers at the moment
Availability: Flexible in my availability
ETA: Immediately
Pictures (optional):N/A

**Comment by ExperimentsInHonesty** [comment]
@KazushiR I see that you are currently on a medium issue.  Congrats!

Its a 1pt issue, so we assume you will be finished in about a week.  If it takes longer, or the next issue that you work on (large), takes longer, you will get a request for update.  At that point you will be able to check off this part of the issue and it will be complete.

- [ ] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report. If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. This update should be done every week for every issue that you are assigned to. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
```
Provide Update
1. Progress
2. Blockers
3. Availability
4. ETA
```

**Comment by jdingeman** [comment]
Hey @KazushiR, hope you're doing well. It looks like you're still needing a medium issue. Are you facing any challenges right now that we can support you with?

**Comment by ExperimentsInHonesty** [comment]
- It looks like you have done an effective update on https://github.com/hackforla/website/issues/2600, so I am going to close this prework issue.

**Comment by HackforLABot** [comment]
## Activity Log: KazushiR
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- KazushiR opened: https://github.com/hackforla/website/issues/2586 at 12/13/2021, 07:44 PM PST
- KazushiR assigned: https://github.com/hackforla/website/issues/2586#issuecomment-993128982 at 12/13/2021, 07:45 PM PST
- KazushiR commented on: https://github.com/hackforla/website/issues/2586#issuecomment-993907307 at 12/14/2021, 11:27 AM PST
- KazushiR closed as completed: https://github.com/hackforla/website/issues/2586#event-5787324050 at 12/17/2021, 03:41 PM PST
- KazushiR assigned: https://github.com/hackforla/website/issues/2437#issuecomment-958242332 at 01/12/2022, 06:47 PM PST
- KazushiR commented on: https://github.com/hackforla/website/issues/2437#issuecomment-1029566209 at 02/03/2022, 05:50 PM PST
- KazushiR opened: https://github.com/hackforla/website/pull/2752 at 02/08/2022, 12:58 PM PST
- KazushiR PR was closed: https://github.com/hackforla/website/pull/2752#event-6032144939 at 02/08/2022, 11:40 PM PST
- KazushiR opened: https://github.com/hackforla/website/pull/2778 at 02/14/2022, 10:03 AM PST
- KazushiR commented on: https://github.com/hackforla/website/pull/2778#issuecomment-1040738189 at 02/15/2022, 12:05 PM PST
- KazushiR commented on: https://github.com/hackforla/website/pull/2778#issuecomment-1040947068 at 02/15/2022, 04:45 PM PST
- KazushiR issue closed by PR 2778: https://github.com/hackforla/website/issues/2437#event-6089371358 at 02/17/2022, 02:56 AM PST
- KazushiR PR was merged: https://github.com/hackforla/website/pull/2778#event-6089371387 at 02/17/2022, 02:56 AM PST
- KazushiR assigned: https://github.com/hackforla/website/issues/2384 at 02/17/2022, 01:21 PM PST
- KazushiR opened: https://github.com/hackforla/website/pull/2790 at 02/18/2022, 11:04 AM PST
- KazushiR commented on: https://github.com/hackforla/website/pull/2790#issuecomment-1045860019 at 02/18/2022, 10:14 PM PST
- KazushiR commented on: https://github.com/hackforla/website/pull/2790#issuecomment-1046125358 at 02/19/2022, 03:42 PM PST
- KazushiR commented on: https://github.com/hackforla/website/pull/2790#issuecomment-1048059410 at 02/22/2022, 09:52 AM PST
- KazushiR commented on: https://github.com/hackforla/website/pull/2790#issuecomment-1048059605 at 02/22/2022, 09:52 AM PST
- KazushiR merged: https://github.com/hackforla/website/pull/2790#event-6119925346 at 02/22/2022, 09:52 AM PST
- KazushiR commented on: https://github.com/hackforla/website/pull/2790#issuecomment-1059716431 at 03/04/2022, 11:59 PM PST
- KazushiR opened: https://github.com/hackforla/website/pull/2944 at 03/09/2022, 10:44 AM PST
- KazushiR merged: https://github.com/hackforla/website/pull/2790#event-6248134270 at 03/15/2022, 10:30 PM PDT
- KazushiR closed: https://github.com/hackforla/website/pull/2944#event-6248136479 at 03/15/2022, 10:30 PM PDT
- KazushiR reopened: https://github.com/hackforla/website/pull/2790#event-6248134270 at 03/15/2022, 10:31 PM PDT
- KazushiR submitted review: https://github.com/hackforla/website/pull/2790#pullrequestreview-914579319 at 03/18/2022, 09:16 AM PDT
- KazushiR opened: https://github.com/hackforla/website/pull/2989 at 03/18/2022, 10:48 AM PDT
- KazushiR closed: https://github.com/hackforla/website/pull/2989#event-6267830321 at 03/18/2022, 10:48 AM PDT
- KazushiR reopened: https://github.com/hackforla/website/pull/2989#event-6267830321 at 03/18/2022, 10:48 AM PDT
- KazushiR closed: https://github.com/hackforla/website/pull/2989#event-6267842497 at 03/18/2022, 10:51 AM PDT
- KazushiR commented on: https://github.com/hackforla/website/pull/2790#issuecomment-1072889481 at 03/18/2022, 04:42 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/pull/2790#issuecomment-1073529456 at 03/20/2022, 11:46 PM PDT
- KazushiR opened: https://github.com/hackforla/website/pull/3002 at 03/22/2022, 04:04 PM PDT
- KazushiR closed: https://github.com/hackforla/website/pull/3002#event-6287314793 at 03/22/2022, 04:04 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/pull/2790#issuecomment-1079852108 at 03/26/2022, 11:41 PM PDT
- KazushiR issue closed by PR 2790: https://github.com/hackforla/website/issues/2384#event-6355881381 at 04/01/2022, 04:56 PM PDT
- KazushiR PR was merged: https://github.com/hackforla/website/pull/2790#event-6355881392 at 04/01/2022, 04:56 PM PDT
- KazushiR assigned: https://github.com/hackforla/website/issues/2961#issuecomment-1078521786 at 04/03/2022, 12:29 PM PDT
- KazushiR unassigned: https://github.com/hackforla/website/issues/2961#issuecomment-1078521786 at 04/03/2022, 12:30 PM PDT
- KazushiR assigned: https://github.com/hackforla/website/issues/2930#issuecomment-1059838228 at 04/03/2022, 12:31 PM PDT
- KazushiR unassigned: https://github.com/hackforla/website/issues/2930#issuecomment-1059838228 at 04/04/2022, 04:04 PM PDT
- KazushiR assigned: https://github.com/hackforla/website/issues/3025#issuecomment-1086562240 at 04/04/2022, 04:10 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/issues/3025#issuecomment-1088102933 at 04/04/2022, 04:11 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/issues/3025#issuecomment-1090738732 at 04/06/2022, 01:19 PM PDT
- KazushiR opened: https://github.com/hackforla/website/pull/3037 at 04/07/2022, 07:31 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/pull/3037#issuecomment-1092384400 at 04/07/2022, 07:34 PM PDT
- KazushiR issue closed by PR 3037: https://github.com/hackforla/website/issues/3025#event-6398173043 at 04/08/2022, 10:38 AM PDT
- KazushiR PR was merged: https://github.com/hackforla/website/pull/3037#event-6398173053 at 04/08/2022, 10:38 AM PDT
- KazushiR assigned: https://github.com/hackforla/website/issues/3023#issuecomment-1085456516 at 04/11/2022, 03:44 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/issues/3023#issuecomment-1097526913 at 04/12/2022, 08:55 PM PDT
- KazushiR opened: https://github.com/hackforla/website/pull/3057 at 04/15/2022, 12:25 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/pull/3057#issuecomment-1100471674 at 04/15/2022, 04:39 PM PDT
- KazushiR PR was closed: https://github.com/hackforla/website/pull/3057#event-6568813545 at 05/08/2022, 09:02 AM PDT
- KazushiR unassigned: https://github.com/hackforla/website/issues/3023#issuecomment-1120444851 at 05/08/2022, 09:06 AM PDT
- KazushiR assigned: https://github.com/hackforla/website/issues/3109#issuecomment-1120023610 at 06/03/2022, 04:14 PM PDT
- KazushiR unassigned: https://github.com/hackforla/website/issues/3109#issuecomment-1120023610 at 06/08/2022, 02:05 PM PDT
- KazushiR issue closed by PR 3664: https://github.com/hackforla/website/issues/2600#event-7710012260 at 10/31/2022, 09:25 PM PDT
- KazushiR assigned: https://github.com/hackforla/website/issues/3882 at 01/26/2023, 07:18 PM PST
- KazushiR assigned: https://github.com/hackforla/website/issues/3892 at 01/30/2023, 04:31 PM PST
- KazushiR commented on: https://github.com/hackforla/website/issues/3892#issuecomment-1409572359 at 01/30/2023, 04:33 PM PST
- KazushiR commented on: https://github.com/hackforla/website/issues/3882#issuecomment-1409573274 at 01/30/2023, 04:34 PM PST
- KazushiR opened: https://github.com/hackforla/website/pull/3905 at 01/30/2023, 07:45 PM PST
- KazushiR commented on: https://github.com/hackforla/website/pull/3905#issuecomment-1411129167 at 01/31/2023, 02:03 PM PST
- KazushiR merged: https://github.com/hackforla/website/pull/3905#event-8406135362 at 01/31/2023, 08:01 PM PST
- KazushiR reopened: https://github.com/hackforla/website/pull/3905#event-8406135362 at 01/31/2023, 08:01 PM PST
- KazushiR PR was merged: https://github.com/hackforla/website/pull/3905#event-8406815140 at 01/31/2023, 10:23 PM PST
- KazushiR issue closed by PR 3905: https://github.com/hackforla/website/issues/3892#event-8406815329 at 01/31/2023, 10:23 PM PST
- KazushiR assigned: https://github.com/hackforla/website/issues/3255#issuecomment-1384677733 at 01/31/2023, 10:46 PM PST
- KazushiR commented on: https://github.com/hackforla/website/issues/3255#issuecomment-1411547994 at 01/31/2023, 10:51 PM PST
- KazushiR opened: https://github.com/hackforla/website/pull/3923 at 02/01/2023, 11:23 PM PST
- KazushiR commented on: https://github.com/hackforla/website/pull/3923#issuecomment-1414751605 at 02/02/2023, 07:01 PM PST
- KazushiR commented on: https://github.com/hackforla/website/issues/3882#issuecomment-1418203854 at 02/05/2023, 10:07 AM PST
- KazushiR commented on: https://github.com/hackforla/website/pull/3930#issuecomment-1418205551 at 02/05/2023, 10:14 AM PST
- KazushiR submitted review: https://github.com/hackforla/website/pull/3930#pullrequestreview-1284598591 at 02/05/2023, 10:14 PM PST
- KazushiR PR was merged: https://github.com/hackforla/website/pull/3923#event-8453667165 at 02/06/2023, 07:12 PM PST
- KazushiR issue closed by PR 3923: https://github.com/hackforla/website/issues/3255#event-8453667284 at 02/06/2023, 07:12 PM PST
- KazushiR assigned: https://github.com/hackforla/website/issues/3848 at 02/06/2023, 07:29 PM PST
- KazushiR commented on: https://github.com/hackforla/website/issues/3848#issuecomment-1420147935 at 02/06/2023, 07:30 PM PST
- KazushiR commented on: https://github.com/hackforla/website/issues/3848#issuecomment-1425187339 at 02/09/2023, 09:18 PM PST
- KazushiR closed as completed: https://github.com/hackforla/website/issues/3882#event-8498768656 at 02/12/2023, 10:15 AM PST
- KazushiR commented on: https://github.com/hackforla/website/issues/3848#issuecomment-1427194194 at 02/12/2023, 05:14 PM PST
- KazushiR commented on: https://github.com/hackforla/website/pull/3995#issuecomment-1434249681 at 02/16/2023, 11:41 PM PST
- KazushiR commented on: https://github.com/hackforla/website/issues/3882#issuecomment-1435396969 at 02/17/2023, 03:37 PM PST
- KazushiR closed as completed: https://github.com/hackforla/website/issues/3882#event-8551788611 at 02/17/2023, 03:37 PM PST
- KazushiR commented on: https://github.com/hackforla/website/issues/3848#issuecomment-1435439351 at 02/17/2023, 05:21 PM PST
- KazushiR submitted review: https://github.com/hackforla/website/pull/3995#pullrequestreview-1304455194 at 02/17/2023, 05:45 PM PST
- KazushiR commented on: https://github.com/hackforla/website/pull/4018#issuecomment-1437914596 at 02/20/2023, 10:14 PM PST
- KazushiR issue closed as completed: https://github.com/hackforla/website/issues/3848#event-8576290487 at 02/21/2023, 05:30 PM PST
- KazushiR assigned: https://github.com/hackforla/website/issues/2306#issuecomment-1250339398 at 02/21/2023, 05:56 PM PST
- KazushiR commented on: https://github.com/hackforla/website/issues/2306#issuecomment-1439328295 at 02/21/2023, 05:56 PM PST
- KazushiR submitted review: https://github.com/hackforla/website/pull/4018#pullrequestreview-1308440419 at 02/21/2023, 06:44 PM PST
- KazushiR commented on: https://github.com/hackforla/website/pull/4021#issuecomment-1439376446 at 02/21/2023, 07:09 PM PST
- KazushiR commented on: https://github.com/hackforla/website/pull/4021#issuecomment-1442504674 at 02/23/2023, 02:18 PM PST
- KazushiR commented on: https://github.com/hackforla/website/pull/4030#issuecomment-1442512937 at 02/23/2023, 02:27 PM PST
- KazushiR commented on: https://github.com/hackforla/website/pull/4030#issuecomment-1442514714 at 02/23/2023, 02:28 PM PST
- KazushiR submitted review: https://github.com/hackforla/website/pull/4021#pullrequestreview-1312244607 at 02/23/2023, 02:33 PM PST
- KazushiR commented on: https://github.com/hackforla/website/issues/2306#issuecomment-1445458787 at 02/26/2023, 12:25 PM PST
- KazushiR commented on: https://github.com/hackforla/website/pull/4058#issuecomment-1449140350 at 02/28/2023, 04:43 PM PST
- KazushiR commented on: https://github.com/hackforla/website/pull/4058#issuecomment-1449191571 at 02/28/2023, 05:45 PM PST
- KazushiR submitted review: https://github.com/hackforla/website/pull/4058#pullrequestreview-1319154391 at 02/28/2023, 11:14 PM PST
- KazushiR commented on: https://github.com/hackforla/website/pull/4123#issuecomment-1458935014 at 03/07/2023, 02:03 PM PST
- KazushiR commented on: https://github.com/hackforla/website/pull/4123#issuecomment-1461053123 at 03/08/2023, 03:59 PM PST
- KazushiR commented on: https://github.com/hackforla/website/issues/2306#issuecomment-1461117277 at 03/08/2023, 05:17 PM PST
- KazushiR submitted review: https://github.com/hackforla/website/pull/4123#pullrequestreview-1333898805 at 03/09/2023, 02:19 PM PST
- KazushiR assigned: https://github.com/hackforla/website/issues/4113#issuecomment-1457327205 at 03/13/2023, 02:15 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/issues/2306#issuecomment-1466973740 at 03/13/2023, 02:16 PM PDT
- KazushiR unassigned: https://github.com/hackforla/website/issues/2306#issuecomment-1466973740 at 03/13/2023, 02:16 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/issues/4113#issuecomment-1466988442 at 03/13/2023, 02:28 PM PDT
- KazushiR unassigned: https://github.com/hackforla/website/issues/4113#issuecomment-1466988442 at 03/13/2023, 03:25 PM PDT
- KazushiR assigned: https://github.com/hackforla/website/issues/2600#issuecomment-1463378139 at 03/13/2023, 03:49 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/issues/2600#issuecomment-1467082947 at 03/13/2023, 03:51 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/pull/4178#issuecomment-1469213366 at 03/14/2023, 07:40 PM PDT
- KazushiR submitted review: https://github.com/hackforla/website/pull/4178#pullrequestreview-1342762968 at 03/15/2023, 07:20 PM PDT
- KazushiR issue closed as completed: https://github.com/hackforla/website/issues/3882#event-8848487333 at 03/26/2023, 05:20 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/pull/4278#issuecomment-1485845334 at 03/27/2023, 01:52 PM PDT
- KazushiR submitted review: https://github.com/hackforla/website/pull/4278#pullrequestreview-1359861925 at 03/27/2023, 01:58 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/pull/4278#issuecomment-1487574669 at 03/28/2023, 01:50 PM PDT
- KazushiR opened: https://github.com/hackforla/website/pull/4339 at 03/28/2023, 11:27 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/pull/4339#issuecomment-1488016700 at 03/28/2023, 11:29 PM PDT
- KazushiR submitted review: https://github.com/hackforla/website/pull/4278#pullrequestreview-1362371368 at 03/28/2023, 11:58 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/pull/4339#issuecomment-1489437271 at 03/29/2023, 03:48 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/pull/4338#issuecomment-1489437593 at 03/29/2023, 03:48 PM PDT
- KazushiR submitted review: https://github.com/hackforla/website/pull/4338#pullrequestreview-1364457014 at 03/30/2023, 12:07 AM PDT
- KazushiR commented on: https://github.com/hackforla/website/pull/4347#issuecomment-1490853946 at 03/30/2023, 12:49 PM PDT
- KazushiR submitted review: https://github.com/hackforla/website/pull/4347#pullrequestreview-1365838399 at 03/30/2023, 12:59 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/pull/4339#issuecomment-1493761542 at 04/02/2023, 11:38 PM PDT
- KazushiR submitted review: https://github.com/hackforla/website/pull/4384#pullrequestreview-1368379257 at 04/02/2023, 11:44 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/pull/4408#issuecomment-1496674079 at 04/04/2023, 03:20 PM PDT
- KazushiR submitted review: https://github.com/hackforla/website/pull/4408#pullrequestreview-1373453399 at 04/05/2023, 11:18 AM PDT
- KazushiR merged: https://github.com/hackforla/website/pull/4339#event-8939240258 at 04/05/2023, 01:37 PM PDT
- KazushiR reopened: https://github.com/hackforla/website/pull/4339#event-8939240258 at 04/05/2023, 01:37 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/pull/4339#issuecomment-1498122973 at 04/05/2023, 01:39 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/pull/4339#issuecomment-1498172184 at 04/05/2023, 02:15 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/pull/4339#issuecomment-1498174902 at 04/05/2023, 02:17 PM PDT
- KazushiR assigned: https://github.com/hackforla/website/issues/3906 at 04/05/2023, 03:58 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/issues/3906#issuecomment-1498259319 at 04/05/2023, 03:59 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/pull/4443#issuecomment-1500793790 at 04/07/2023, 10:10 PM PDT
- KazushiR submitted review: https://github.com/hackforla/website/pull/4443#pullrequestreview-1376875053 at 04/09/2023, 01:17 AM PDT
- KazushiR commented on: https://github.com/hackforla/website/pull/4444#issuecomment-1501073345 at 04/09/2023, 01:18 AM PDT
- KazushiR submitted review: https://github.com/hackforla/website/pull/4472#pullrequestreview-1380312801 at 04/11/2023, 07:04 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/pull/4468#issuecomment-1504439526 at 04/11/2023, 07:13 PM PDT
- KazushiR opened: https://github.com/hackforla/website/issues/4478 at 04/11/2023, 08:01 PM PDT
- KazushiR submitted review: https://github.com/hackforla/website/pull/4468#pullrequestreview-1384123425 at 04/13/2023, 12:59 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/issues/3906#issuecomment-1507833861 at 04/13/2023, 07:14 PM PDT
- KazushiR unassigned: https://github.com/hackforla/website/issues/3906#issuecomment-1507833861 at 04/13/2023, 07:14 PM PDT
- KazushiR assigned: https://github.com/hackforla/website/issues/4296 at 04/13/2023, 07:18 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/issues/4296#issuecomment-1507836289 at 04/13/2023, 07:18 PM PDT
- KazushiR submitted review: https://github.com/hackforla/website/pull/4554#pullrequestreview-1398425637 at 04/24/2023, 09:40 AM PDT
- KazushiR commented on: https://github.com/hackforla/website/issues/4296#issuecomment-1520500214 at 04/24/2023, 09:41 AM PDT
- KazushiR commented on: https://github.com/hackforla/website/issues/4296#issuecomment-1522594701 at 04/25/2023, 05:34 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/issues/4296#issuecomment-1522665101 at 04/25/2023, 07:16 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/issues/4296#issuecomment-1526887214 at 04/27/2023, 07:22 PM PDT
- KazushiR commented on: https://github.com/hackforla/website/issues/4296#issuecomment-1547272420 at 05/14/2023, 11:38 PM PDT
- KazushiR unassigned: https://github.com/hackforla/website/issues/4296 at 05/14/2023, 11:41 PM PDT
